- a/node_modules/ai/rsc/dist/index.js
+++ b/node_modules/ai/rsc/dist/index.js
@@ -1,7 +1,7 @@
 'use strict';
 
 if (process.env.NODE_ENV === 'development') {
-  module.exports = require('./index.dev.js');
+  module.exports = require('./index.js');
 } else {
   module.exports = require('./index.js');
 }
